build/change-version.js

```diff
@@ -34,11 +37,17 @@ function regExpQuoteReplacement(string) {
   return string.replace(/\$/g, '$$')
 }
 
-async function replaceRecursively(file, oldVersion, newVersion) {
+async function replaceRecursively(file, oldVersion, newVersion, limitNumberOfReplacements = 0) {
   const originalString = await fs.readFile(file, 'utf8')
-  const newString = originalString.replace(
-    new RegExp(regExpQuote(oldVersion), 'g'), regExpQuoteReplacement(newVersion)
-  )
+  let newString = originalString
+
+  if (limitNumberOfReplacements > 0) {
+    for (let i = 0; i < limitNumberOfReplacements; i++) {
```

Comment:
Can you try changing this to `for of`?

Reply:
Yes I try ASAP and let you know.

Reply:
Can do a `for (let i = limitNumberOfReplacements; i--;) {` as in some JS components. Note sure about a `for of`.

Reply:
For of is preferable, but if it's not possible, let's keep the plain old loop :)

Alternatives are `map` or even `reduce` (but we might need to tweak linting in case of reduce).

Reply:
@julien-deramond can you take care of the above comment please? I think we can land the PR after that.

Reply:
Sorry @XhmikosR I'm in holidays for a couple of weeks with just a phone. From what I remember `for of` wasn't possible so I'd say let's keep the plain old loop if it sounds good to you. If needed I still can try to find another way of doing it when I'm back.